services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -402,15 +368,16 @@ public boolean generateVMSetupCommand(Long ssAHostId) {
         String copyPasswd = _configDao.getValue("secstorage.copy.password");
         setupCmd.setCopyPassword(copyPasswd);
         setupCmd.setCopyUserName(TemplateConstants.DEFAULT_HTTP_AUTH_USER);
+
         Answer answer = _agentMgr.easySend(ssAHostId, setupCmd);
         if (answer != null && answer.getResult()) {
             if (s_logger.isDebugEnabled()) {
-                s_logger.debug("Successfully programmed http auth into " + secStorageVm.getHostName());
+                s_logger.debug(String.format("Successfully created http auth into secondary storage VM [%s].", hostName));
```

Comment:
IMHO `programmed` can be indeed changed to something that best describes the logged operation (set the allowed internal sites, username, and password).

`Created` is an option, but I think that `set` fits better.

I would suggest changing from: `Successfully created http auth into secondary storage VM [s-123-VM]`
To: `Successfully set HTTP auth into secondary storage VM [s-123-VM]`

Reply:
@GabrielBrascher done.